scripts/get_zeton_token.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+command_exists() {
+  command -v "$1" &> /dev/null
+}
+
+if ! command_exists curl; then
+  echo "'curl' is not installed. Please install 'curl' to use this script."
+  exit 1
+fi
+
+if ! command_exists jq; then
+  echo "'jq' is not installed. Please install 'jq' to use this script."
+  exit 1
+fi
+
+copy_to_clipboard() {
+  if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    if command_exists xclip; then
+      echo "$1" | xclip -selection clipboard
+      echo "Token has been copied to clipboard using xclip!"
+    else
+      echo "'xclip' not found. Token: $1"
+    fi
+  elif [[ "$OSTYPE" == "darwin"* ]]; then
+    echo "$1" | pbcopy
+    echo "Token has been copied to clipboard using pbcopy!"
+  else
+    echo "Unsupported OS. This script works on GNU/Linux and MacOS only."
+  fi
+}
+
+TOKEN=$(curl -s -X 'POST' \
+  'http://localhost:8000/api/token-auth/' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "username": "opiekun1",
+  "password": "opiekun1"
```

Comment:
In theory, we shouldn't put hardcoded credentials in the scripts, but since they are part of our fixtures, I think that's fine. However, I'd put them in a separate variable, make `opiekun1` a default and allow the user to override it through the command line arguments

Reply:
Exactly, I saw those strings checked in to the repo already so I decided to use them in the script. I applied your suggesion